### PR TITLE
Add Eltex platform with 8 templates

### PR DESCRIPTION
**Pull request by PavloSkliarenko**
Templates for commands:

`show interface`
`show interfaces description`
`show interfaces status`
`show interfaces switchport`
`show ip interface`
`show system`
`show system id`
`show vlan`

### CODE REVIEW

File: ntc_templates/templates/eltex_show_interfaces_description.textfsm [hunk 1]
```diff
@@ -0,0 +1,73 @@
+Value PORT ((?!Port|Ch|-)\S+)
+Value PORT_MODE (Trunk|Access(?:\s*\(\d+\))?|Customer(?:\s*\(\d+\))?)
```
Comment (jvanderaa) [inline ntc_templates/templates/eltex_show_interfaces_description.textfsm:2]:
```suggestion
Value INTERFACE ((?!Port|Ch|-)\S+)
Value INTERFACE_MODE (Trunk|Access(?:\s*\(\d+\))?|Customer(?:\s*\(\d+\))?)
```

File: ntc_templates/templates/eltex_show_interfaces_status.textfsm [hunk 2]
```diff
@@ -0,0 +1,69 @@
+Value PORT (\S+)
```
Comment (jvanderaa) [inline ntc_templates/templates/eltex_show_interfaces_status.textfsm:1]:
```suggestion
Value INTERFACE (\S+)
```

File: ntc_templates/templates/eltex_show_interfaces_switchport.textfsm [hunk 3]
```diff
@@ -0,0 +1,88 @@
+Value PORT (.*?)
+Value PORT_MODE (.*?)
```
Comment (jvanderaa) [inline ntc_templates/templates/eltex_show_interfaces_switchport.textfsm:2]:
```suggestion
Value INTERFACE (.*?)
Value INTERFACE_MODE (.*?)
```

File: ntc_templates/templates/eltex_show_ip_interface.textfsm [hunk 4]
```diff
@@ -0,0 +1,53 @@
+Value GATEWAY_IP (\S+)
+Value GATEWAY_STATUS (\S+)
+Value GATEWAY_TYPE (\S+)
+Value IP (\S+)
+Value I_F (.*?)
```
Comment (jvanderaa) [inline ntc_templates/templates/eltex_show_ip_interface.textfsm:5]:
```suggestion
Value I_F (.*?)
```

Is that an interface? Let's update this.

File: ntc_templates/templates/eltex_show_system.textfsm [hunk 5]
```diff
@@ -0,0 +1,73 @@
+Value DESCRIPTION (.*)
+Value UP_TIME (.*)
```
Comment (jvanderaa) [inline ntc_templates/templates/eltex_show_system.textfsm:2]:
```suggestion
Value UPTIME (.*)
```

File: ntc_templates/templates/eltex_show_system_id.textfsm [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Value UNIT (\d+)
+Value MAC ([a-zA-Z0-9]{2}(?::[a-zA-Z0-9]{2}){5})
```
Comment (jvanderaa) [inline ntc_templates/templates/eltex_show_system_id.textfsm:2]:
```suggestion
Value MAC_ADDRESS ([a-zA-Z0-9]{2}(?::[a-zA-Z0-9]{2}){5})
```

File: ntc_templates/templates/eltex_show_vlan.textfsm [hunk 7]
```diff
@@ -0,0 +1,40 @@
+Value Required VLAN_ID (\d+)
+Value NAME (\S+)
```
Comment (jvanderaa) [inline ntc_templates/templates/eltex_show_vlan.textfsm:2]:
```suggestion
Value VLAN_NAME (\S+)
```

**Review by jvanderaa — CHANGES_REQUESTED** [review]

File: ntc_templates/templates/eltex_show_ip_interface.textfsm (hunk 4 above)
Comment (PavloSkliarenko) [reply ntc_templates/templates/eltex_show_ip_interface.textfsm:5]: Yes. This is interface. I'll fix it

**Comment by PavloSkliarenko** [comment]
Fix value names according to new Data model

**Review by jvanderaa — APPROVED** [review]

**Pull request merged**